querydsl-sql/pom.xml

```diff
@@ -259,6 +259,7 @@
                 <value>target/derby.log</value>
             </property>
           </systemProperties>
+          <argLine>-Duser.timezone=UTC</argLine>
```

Comment:
Apparently in java 8 setting `user.timezone` in `systemProperties` results in it getting read too late and being cached as the system default. Using `argLine` instead works around that.